src/psyclone/gocean1p0.py

```diff
@@ -703,6 +706,10 @@ def add_bounds(bound_info):
             # Now replace any {start}/{stop} expression in the loop
             # with a valid integer value:
             do_string = do_string.format(start='15', stop='25')
+            # We need to make sure the fparser is properly initialised, which
```

Comment:
I think this fix is the right one. Please could you move the parser creation up out of this loop?

Reply:
Oops, I never looked further up 😊